packages/http-client-java/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to TypeSpec Java Emitter Library
+
+## Prerequisite
+
+Install [Node.js](https://nodejs.org/) 20 or above. (Verify by running `node --version`)
+
+Install [Java](https://docs.microsoft.com/java/openjdk/download) 17 or above. (Verify by running `java --version`)
+
+Install [Maven](https://maven.apache.org/download.cgi). (Verify by running `mvn --version`)
+
+## Build and Pack
```

Comment:
Before `Build and Pack`, do we want to mention dev process, e.g. the folder structure for emitter and generator?

Reply:
Added a short "Project Structure" section.